io.github.dummerle.rare.json

```
@@ -0,0 +1,48 @@
+{
+    "app-id": "io.github.dummerle.rare",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/rare",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
```

```suggestion
        "--socket=fallback-x11",
```

assuming wayland support

---

This does not work. Qt raises an exception